### ISSUE TITLE
Marvin.IDP is not starting in final sample 

### ISSUE THREAD

**Issue by dgxhubbard** [KevinDockx/SecuringAspNetCore6WithOAuth2AndOIDC#1]
When I try to startup up Marvin.IDP in Finished Sample there is an exception:

Exception thrown: 'Azure.Identity.CredentialUnavailableException' in Azure.Security.KeyVault.Secrets.dll

which causes the IDP to shut down immediately

**Comment by KevinDockx** [comment]
Yep, that's expected behavior: the IDP stores secrets in Azure Keyvault, and those are not distributed together with the codebase.  If you're following along with the course, you'll learn how to do that in the final module.  If not, you can comment out the relevant parts (this line & following: https://github.com/KevinDockx/SecuringAspNetCore6WithOAuth2AndOIDC/blob/main/Finished%20sample/Marvin.IDP/HostingExtensions.cs#L34) and run the IDP with the developer signing credentials (.AddDeveloperSigningCredential).  

Hope this helps!
